pyproject.toml

```diff
@@ -27,12 +27,7 @@ classifiers = [
     "Topic :: Scientific/Engineering :: Image Processing",
 ]
 requires-python = ">=3.8"
-dependencies = [
-    "imagecodecs",
-    "napari>=0.4.18, <0.5.0",
-    "numpy",
-    "dask-image",
-]
+dependencies = ["imagecodecs", "napari>=0.4.18, <0.5.0", "numpy", "dask-image"]
```

Comment:
One of the linters insists on this. Might be an idea to add explanatory comments.